runway/cfngin/hooks/utils.py

```diff
@@ -5,6 +5,11 @@
 import sys
 
 from runway.util import load_object_from_string
+# from runway.variables import Variable, resolve_variables
+
+from ..exceptions import FailedVariableLookup
+# TODO replace after https://github.com/onicagroup/runway/pull/116
```

Comment:
@ITProKyle merged ^ and ready for updating

Reply:
updated